pkg/operator/operator.go

```diff
@@ -210,10 +210,20 @@ func (optr *Operator) maoConfigFromInstallConfig() (*OperatorConfig, error) {
 		return nil, err
 	}
 
+	// TODO: Remove once we transition over machine.openshift.io group
+	var providerDreprecatedControllerImage string
+	switch provider {
+	case AWSProvider:
+		providerDreprecatedControllerImage = images.ClusterAPIControllerAWSDeprecated
```

Comment:
Our first pass at landing this work continues to use the existing images, correct?

Reply:
That's correct. Once we merge this:
- k8s-e2e can go green for https://github.com/openshift/cluster-api-provider-aws/pull/147 since it needs the new nodelink controller included in this PR
e2e-aws/e2e-aws-operator can go green here https://github.com/openshift/cluster-api-provider-aws/pull/147 since the deprecated controllers are still run by this changes